types/images.d.ts

```diff
@@ -0,0 +1,46 @@
+type StaticImageData = {
+  src: string;
+  height: number;
+  width: number;
+  placeholder?: string;
+};
+
+declare module "*.png" {
+  const content: StaticImageData;
+  export default content;
+}
+
+declare module "*.svg" {
```

Comment:
To make a very long story short, we need to override the definition for `.svg` files because nextjs treats them as `any` by default. This is wrong for us, because we use `svgr` to import and use svgs as react components.